packages/server/src/hub/handlers/handle-ongoing-process-action.ts

```diff
@@ -132,12 +138,19 @@ async function handleCommitmentsReceived(action: CommitmentsReceived) {
       ledgerCommitmentRound,
       currentCommitment && asConsensusCommitment(currentCommitment),
     );
-    return communication.sendCommitmentsReceived(
-      nextParticipant,
-      processId,
-      [...incomingCommitments, { commitment, signature: (signature as unknown) as string }],
-      action.protocolLocator,
-    );
+    const response = [];
+    const otherParticipants = participants.filter(p => participants.indexOf(p) !== ourIndex);
+    for (const participant of otherParticipants) {
```

Comment:
This can be
```
const response = participants
  .filter((_, idx) => idx !== ourIndex)
  .map(p => communication.sendCommitmentsReceived(...))
```